care/facility/management/commands/clean_patient_phone_numbers.py

```diff
@@ -0,0 +1,38 @@
+import json
+from typing import Optional
+
+import phonenumbers
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'phonenumbers'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5332931)